src/SIS_dyn_cgrid.F90

```diff
@@ -996,6 +1029,8 @@ subroutine SIS_C_dynamics(ci, mis, mice, ui, vi, uo, vo, &
       v2_at_u =  CS%drag_bg_vel2 + 0.25 * &
                      (((vi(i,J)-vo(i,J))**2 + (vi(i+1,J-1)-vo(i+1,J-1))**2) + &
                       ((vi(i+1,J)-vo(i+1,J))**2 + (vi(i,J-1)-vo(i,J-1))**2))
+      if (CS%lemieux_landfast) v2_at_u_min = &
+                     min(vi(i,J), vi(i+1,J-1), vi(i+1,J), vi(i,J-1))**2
```

Comment:
If these are all positive, this picks out the minimum magnitude, but if these are all negative this picks out the largest magnitude.  Maybe there should be absolute values to these velocities, or they should be squared before taking the minimum.